internal/manifests/queryfrontend/query_frontend.go

```diff
@@ -90,13 +90,16 @@ func BuildQueryFrontend(params manifestutils.Params) ([]client.Object, error) {
 			jaegerUIAuthentication := tempo.Spec.Template.QueryFrontend.JaegerQuery.Authentication
 
 			if jaegerUIAuthentication != nil && jaegerUIAuthentication.Enabled {
+				defaultOauthProxyResources := manifestutils.Resources(tempo, manifestutils.QueryFrontendComponentName, tempo.Spec.Template.QueryFrontend.Replicas)
```

Comment:
I think it should take the resources from `jaeger-frontend-proxy`, not from `manifestutils.QueryFrontendComponentName`, or am I reading it wrong?

Reply:
Oh true! a mistake! good catch thanks